mock-data/connectPost.js

```diff
@@ -0,0 +1,20 @@
+const pg = require('pg');
+const config  = {
+	database: 'booking',
```

Comment:
Your indentations are 4 spaces I think. Check your settings and try changing the tab size = 2 spaces.